lib/sles4sap/ipaddr2.pm

```diff
@@ -60,11 +71,30 @@ our $nat_pub_ip = DEPLOY_PREFIX . '-nat_pub_ip';
 # Storage account name must be between 3 and 24 characters in length
 # and use numbers and lower-case letters only.
 our $storage_account = DEPLOY_PREFIX . 'storageaccount';
-our $priv_ip_range = '192.168.';
-our $frontend_ip = $priv_ip_range . '0.50';
+our %priv_net_address_range = get_private_ip_range();
+our $priv_ip_range = $priv_net_address_range{priv_ip_range};
+our $frontend_ip = $priv_ip_range . '.50';
 our $ping_cmd = 'ping -c 3';
 our $key_id = 'id_rsa';
 
+=head2 get_private_ip_range
+
+    my $range = get_private_ip_range();
+
+count the private ip range and return
+=cut
+
+sub get_private_ip_range {
+    my %range = (vnet_address_range => '192.168.0.0/16', subnet_address_range => '192.168.0.0/24');
+    if (my $worker_id = get_var("WORKER_ID")) {
```

Comment:
I have observation about:
```
my %range = (vnet_address_range => '192.168.0.0/16', subnet_address_range => '192.168.0.0/24');
    if (my $worker_id = get_var("WORKER_ID")) {
        %range = qesap_az_calculate_address_range(slot => $worker_id);
    }
```

as WORKER_ID  is always defined => in any kind of execution of this code will enter in the if
[4:52](https://suse.slack.com/archives/D06R1GMFD53/p1753368753415699)
right?
4:54
here an example to proof it https://openqaworker15.qa.suse.cz/tests/333957/logfile?filename=serial_terminal.txt#line-126 has 10.4.8.0/21